README.md

```diff
@@ -232,6 +232,9 @@ for any of its variants), and want to avoid a small amount of overhead, you can
 Unless you have data showing that the match statement generated in the `try_from` above is a bottleneck for you,
 you should avoid doing this, as the unsafe code has potential to cause serious memory issues in your program.
 
+Note that this derive ignores any `default`, `catch_all`, and `alternatives` attributes on the enum.
+If you need support for conversions from these values, you should use `TryFromPrimitive` or `FromPrimitive`.
+
```

Comment:
Just to be extra cautious, let's add a (counter-)example:

```suggestion
Note that this derive ignores any `default`, `catch_all`, and `alternatives` attributes on the enum.
If you need support for conversions from these values, you should use `TryFromPrimitive` or `FromPrimitive`.

  - This means, for instance, that the following is UB:

    ```rust ,no_run
    #[derive(UnsafeFromPrimitive)]
    #[repr(u8)]
    enum Number {
        Zero = 0,

        // same for `#[num_enum(catch_all)]`, and `#[num_enum(alternatives = [2, ...])]`
        #[default]        
        One = 1,
    }
    let _undefined_behavior = unsafe { Number::from_unchecked(2) };
    ```
```